docs/examples/tutorial/string/utf_eight.pyx

```diff
@@ -1,15 +1,20 @@
+# cython: language_level=3
```

Comment:
I don't think we should have these any more. It's the default now.

Reply:
fixed.